Makefile

```diff
@@ -135,16 +135,13 @@ lint-node: compile-node
 lint-samtemplates:
 	poetry run cfn-lint -I "SAMtemplates/**/*.yaml" 2>&1 | grep "Run scan"
 
-lint-python:
-	poetry run flake8 scripts/*.py --config .flake8
-
 lint-githubactions:
 	actionlint
 
 lint-githubaction-scripts:
 	shellcheck .github/scripts/*.sh
 
-lint: lint-node lint-samtemplates lint-python lint-githubactions lint-githubaction-scripts
+lint: lint-node lint-samtemplates lint-githubactions lint-githubaction-scripts
```

Comment:
we should remove the last vestige of python as well: [here](https://github.com/search?q=repo%3ANHSDigital%2Fprescriptionsforpatients++language%3APython&type=code)

Reply:
`calculate_version.py` was removed as part of d0b129639b5721916258f7d126946b25c3d6ba01